src/components/ResultsFilter/ResultsFilter.jsx

```diff
@@ -1,44 +1,33 @@
-import React, { useState } from "react";
-
+import React from "react";
+import { Nav } from 'react-bootstrap';
 import './ResultsFilter.scss';
 
-const ResultsFilter = ({ filterHandle }) => {
-  
-  const [appState, setState] = useState({
-    activeState: 'All'
-  });
-
-  const menuItems = [
-    'All',
-    'Documentary',
-    'Comedy',
-    'Horror',
-    'Crime'
-  ]
+const noop = () => {};
   
-  const toggleActive = (index) => { 
-    setState({activeState: menuItems[index]});
-  }
+const menuItems = [
+  'All',
+  'Documentary',
+  'Comedy',
+  'Horror',
+  'Crime'
+]
 
-  const toggleActiveClass = (index) => {
-    const classname = "menu-item";
-    if(menuItems[index] === appState.activeState) {
-      return classname + " active";
-    } else {
-      return classname;
-    }
+const ResultsFilter = ({ filmState = [], onFilterChange = noop }) => {
+ 
+  const onClick = (e) => {
+    onFilterChange(e.target.innerText);
```

Comment:
why you used innerText ? maybe better e.target.value?

Reply:
Because I don't have such field, maybe it's in bootstrap components only